docs/core/compatibility/sdk/8.0/istrimmable-netstandard-libraries.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Breaking change: IsTrimmable warns when used with netstandard libraries"
+description: Learn about a breaking change in the .NE 8 SDK where netstandard libraries that specify IsTrimmable produce a new warning.
+ms.date: 08/21/2023
+---
+# IsTrimmable warns when used with netstandard libraries
+
+Libraries that set `<IsTrimmable>true</IsTrimmable>` with a `TargetFramework` that is any version of netstandard produce a warning that `IsTrimmable` is unsupported.
+Setting `<EnableTrimAnalyzer>true</EnableTrimAnalyzer>` in such a library produces the same warning.
```

Comment:
I thought the warning is also issued when the library targets NetFX or old .NET Core TFMs, not just netstandard. I think it's OKish to keep the title simple, but the description should be precise - if it's still the case.